# DetectedObjects

std_msgs/Header header
DetectedObjectMsg[] detected_objects_msgs